test/inference.jl

```diff
@@ -494,3 +494,9 @@ immutable MyType18457{T,F,G}<:AbstractMyType18457{T,F,G} end
 tpara18457{I}(::Type{AbstractMyType18457{I}}) = I
 tpara18457{A<:AbstractMyType18457}(::Type{A}) = tpara18457(supertype(A))
 @test tpara18457(MyType18457{true}) === true
+
+# Issue 20067
+@generated function foo20067{T <: Tuple}(::Type{T}, elements::Tuple)
+           :(T)
+end
+code_typed(DevNull, foo20067, (DataType, Tuple{Int, Int}))
```

Comment:
There is no 3-argument `code_typed`. Unfortunately this test failure was masked by #20027.
It would also be better if there were a `@test` in here.